spec/till_spec.rb

```diff
@@ -0,0 +1,53 @@
+describe 'Till' do
+  let(:till){Till.new Receipt}
+
+  context 'created with' do
+    it 'a total of 0' do
+      expect(till.total).to eq 0
+    end
+
+    it 'a tax total of 0' do
+      expect(till.tax_total).to eq 0
+    end
+
+    it 'an emptly list of orders' do
+      expect(till.ordered_items.empty?).to be true
+    end
+
+    it 'a tax rate of 8.64%' do
+      expect(Till::TAX).to eq 0.0864
+    end
+  end
+
+  context 'product price' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.